base/customize.sh

```diff
@@ -167,20 +147,12 @@ fi
   chcon -R u:object_r:system_file:s0 "$F_TARGETDIR"
   chmod -R 755 "$F_TARGETDIR"
 
-  $UNZIP -qq -o "$ZIPFILE" "files/frida-server-$F_ARCH" -j -d "$F_TARGETDIR"
-  mv "$F_TARGETDIR/frida-server-$F_ARCH" "$F_TARGETDIR/frida-server"
+  mv "$MODPATH/files/frida-server-$F_ARCH" "$F_TARGETDIR/frida-server"
+  rm -rf "$MODPATH/files"
```

Comment:
Do we really need to clean this up? If Magisk extracts it automatically, I would have thought it will also clean it up automatically. I am very reluctant to add a `rm -rf` in the module code.

Reply:
Actually never mind, this seems like it extracts to the permanent installation directory. Can we just drop the `-f` then, so it's a bit safer? `rm -r "$MODPATH/files"`.

Reply:
And also, I can see that auto extraction is the behavior for Magisk, but what about APatch and KernelSU? I have no idea how they work, maybe @enovella can advise.

Reply:
Cleaning is not important, mainly for compatibility and space saving.

Reply:
All three mainstream platforms are compatible with this installation method.